src/drivers/imx/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_local_sources(sof
 	interrupt.c
```

Comment:
We could cleanup commit message, be deleting DO NOT MERGE and this commit requires the cleanup. Also, would be nice to add a link to a RM containing SDMA bits. Maybe link to 8MQ RM.

Reply:
That's a reminder for myself that I need to clean up some bits from this commit. We have seen how unreliable direct links for the RM are so I will look into how to make a reasonably-persistent link for future reference.